### ISSUE TITLE
Documentation page is not mobile-friendly

### ISSUE THREAD

**Issue by cmmartti** [PokeAPI/pokeapi.co#9]
It's fine for now because not many people will be looking up documentation on their `phones, but on smaller viewports I'd like to swap out the table of contents on the left for a stickied breadcrumbs menu bar. The breadcrumbs would show the current page section (such as "Items > Item Categories"), and clicking on each crumb would open a list of links to the crumb's sibling sections.

Working on this now.

**Comment by Naramsim** [comment]
We can drop the description of the tables
```css
thead>tr>th:nth-child(2) {
    display: none;
}

tbody>tr>td:nth-child(2) {
    display: none;
}
```

Get rid of the `display: grid` and leave the TOC on top, or replace it with a search bar.

![pokeapi co_about html iphone 6_7_8](https://user-images.githubusercontent.com/8996268/46137847-bd0e4e80-c24a-11e8-9e41-b00917a97250.png)

**Comment by cmmartti** [comment]
The description is important information, in my opinion, and it would be kind of confusing for it to be missing. I did some experimenting with programmatically adding [`<wbr />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr) tags after underscores and between words in types so they can break onto new lines, and it seems to work pretty well. Also, the breadcrumbs work well too, though they don't work with the keyboard yet so they're not accessible. I'm still working on that.

See the screenshot:

![docs-toc](https://user-images.githubusercontent.com/6448223/46164230-783ee380-c25b-11e8-9a58-89a1a0db4890.png)

Also, I switched from `grid` to `flex` on non-mobile viewports because FlexBox has more support. I could do it with floats too, but I hate mis-using floats like that. The sticky elements use `position: sticky`, which IE doesn't support, but that's okay.
